### PR TITLE
Team3 [1pt] Create basic structure for Dragons - Setup

### PR DESCRIPTION
**Pull request by obibaadoma**
Create a route and a view component. Use <NavLink /> for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
Create a directory for all Redux state slice files.

### CODE REVIEW

**Review by ahmidou09 — COMMENTED** [review]
Hi, @obibaadoma  👋,

Unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request






please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @ahmidou09  in your question so I can receive the notification.

File: src/components/navBar.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          <li key={link.id}>
+            <NavLink className="Nav-bar-element" to={link.path}>{link.text}</NavLink>
+          </li>
+        ))}
+      </div>
+    </nav>
+  );
+};
+
+export default Navbar;
```
Comment (victormc13) [inline src/components/navBar.js]: It is no necessary to make the navbar again, that part of the project is done, don't worry about it, so maybe you could delete the navBar.js file

File: src/redux/dragons/dragons.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        };
+      });
+      return newState;
+    },
+  },
+
+  extraReducers: {
+    [loadDragons.fulfilled]: (state, action) => action.payload,
+  },
+});
```
Comment (victormc13) [inline src/redux/dragons/dragons.js]: Here your `state` parameter is defined but never used it.

File: src/components/Dragons/Dragons.js [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+            src={dragon.img}
+            type={dragon.type}
+            value={dragon.value}
+          />
+        ))}
+
+      </div>
+    </>
+  );
+};
```
Comment (victormc13) [inline src/components/Dragons/Dragons.js]: Here there is some errors related to the Dragons component when trying to make the map of something that is undefined

image

**Review by victormc13 — CHANGES_REQUESTED** [review]
# Changes Required ♻️⚒️  

Hi @obibaadoma 

Great work on making the project requirements 👏! I really appreciate the effort you put in!  **_However, there are few things that we can to polish. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the structure
- ✅ Great job with name variables
- ✅ Correct Linters

## Required Changes ♻️ 

_Please check the comments under the review._

## Optional suggestions ❕
Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @victormc13  in your question so I can receive the notification.

File: src/components/navBar.js (hunk 1 above)
Comment (obibaadoma) [reply src/components/navBar.js]: okay well noted, will take that out of the code

File: src/redux/dragons/dragons.js (hunk 2 above)
Comment (obibaadoma) [reply src/redux/dragons/dragons.js]: will remove that, noted

File: src/components/Dragons/Dragons.js (hunk 3 above)
Comment (obibaadoma) [reply src/components/Dragons/Dragons.js]: done

**Review by ahmidou09 — APPROVED** [review]
Hi @obibaadoma  , 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

a




## Highlights
- Layout matched the given wire frames :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**